warn/scrapers/wv.py

```diff
@@ -0,0 +1,90 @@
+from pathlib import Path
```

Comment:
The WV scraper shouldn't be in the HI PR. Can we pull this out?

Reply:
Sure!